src/oracledb.go

```diff
@@ -20,24 +21,30 @@ type argumentList struct {
 	IsSysDBA        bool   `default:"false" help:"Is the user a SysDBA"`
 	IsSysOper       bool   `default:"false" help:"Is the user a SysOper"`
 	Hostname        string `default:"127.0.0.1" help:"The OracleDB connection host name"`
+	Tablespaces     string `default:"" help:"JSON Array of Tablespaces to collect. If empty will collect all tablespaces."`
 	Port            string `default:"1521" help:"The OracleDB connection port"`
 	ExtendedMetrics bool   `default:"false" help:"Enable extended metrics"`
 }
 
 const (
 	integrationName    = "com.newrelic.oracledb"
-	integrationVersion = "0.1.1"
+	integrationVersion = "0.1.2"
 )
 
 var (
-	args argumentList
+	args                argumentList
+	tableSpaceWhiteList []string
```

Comment:
Is there a reason tableSpaceWhiteList is a global variable rather than something that is passed to the collector?

Reply:
I made it a global variable for two reasons:

1. It's basically an input argument that just needs additional parsing. So I kept it global like the other arguments.
2. It's used only in table space collection and I didn't want to modify the function signature for an `oracleMetricGroup` just to fit a specific case.

If you have a cleaner solution let me know.